### DISCUSSION TITLE
关于 Input 组件开发相关

### DISCUSSION THREAD

**Discussion by Tyh2001** [FightingDesign/fighting-design#61] [Ideas]
# Input 输入框

## Attributes

- `v-model` 绑定值
- `type` 类型，可选值 `text / password`
- `size` 不同尺寸 `large / medium / small / mini`
- `clear` 是否可清空
- `suffix-icon` 后缀 `icon`
- `prefix-icon` 前缀 `icon`
- `disabled` 禁用状态
- `show-pass` 是否展示可以查看密码的按钮
- `autofocus` 是否自动获取焦点
- `name` 原生 `name` 属性
- `width` 自定义宽度（默认 100%）
- `placeholder` 占位符
- `readonly` 是否只读状态
- `max` 最大输入
- `min` 最小输入
- `round` 圆角输入框
- `background` 输入框背景色
- `text-color` 文字颜色
- `border-color` 边框颜色
- `bordered` 是否显示边框

## Event

- `clear` 点击清空按钮触发的回调	
- `enter` 按下回车时候触发的回调
- `change` 在输入框发生变化时候触发的回调
- `onfocus` 获取焦点触发的回调	
- `onblur` 失去焦点触发的回调

## 设计稿

https://www.figma.com/file/JUkONXFsWIaE04cgZbAFBC/Input-%E8%BE%93%E5%85%A5%E6%A1%86?node-id=0%3A1

## 参考文档

可以通过下面文档进行参考设计组件

https://developer.mozilla.org/zh-CN/docs/Web/HTML/Element/Input

**Comment by Tyh2001** [comment]
https://github.com/wmasfoe 进行开发

**Reply by wmasfoe** [reply]
预计这周开发完😘

**Reply by Tyh2001** [reply]
> 预计这周开发完😘

进度如何？
